backend/worker/requirements.txt

```diff
@@ -5,9 +5,10 @@ click==8.1.3
 dateparser==1.1.8
 dnstwist==20230509
 docopt==0.6.2
+git+https://github.com/LeapBeyond/scrubadub.git@d0e12c5d922631af3532d044196b05fb1b7c8c1c
+git+https://github.com/mitmproxy/mitmproxy@e0e46f4
```

Comment:
Outside the scope of this PR, but it would be more consistent if you standardized on which hash identifier to use in this file (either full or partial), as opposed to some of each.

Reply:
See https://github.com/cisagov/XFD/issues/95